cloud/docker/docker.py

```diff
@@ -1110,6 +1127,15 @@ def get_differing_containers(self):
                 self.reload_reasons.append('volumes_from ({0} => {1})'.format(actual_volumes_from, expected_volumes_from))
                 differing.append(container)
 
+            # LOG_DRIVER
+
+            expected_log_driver = set(self.module.params.get('log_driver') or [])
+            actual_log_driver = set(container['HostConfig']['LogConfig']['Type'] or [])
+            if actual_log_driver != expected_log_driver:
```

Comment:
On point 2 :point_up:, there are actually a *lot* of false positives like this for options, especially if you use `state=reloaded` against docker containers that were launched with something other than Ansible. I hit a bunch of other ones this morning (I remember `net` was one).